Doc/library/typing.rst

```diff
@@ -3355,7 +3424,7 @@ Aliases to built-in types
    Deprecated alias to :class:`builtins.set <set>`.
 
    Note that to annotate arguments, it is preferred
-   to use an abstract collection type such as :class:`AbstractSet`
+   to use an abstract collection type such as :class:`~collections.abc.Set`
```

Comment:
It might be worth giving the fully qualified name for this one, as it's quite confusing that it has the same name as `typing.Set` (the entry for which we're in right now), but represents a very different type (an abstract set, equivalent to `typing.AbstractSet` rather than `typing.Set` or `builtins.set`)

```suggestion
   to use an abstract collection type such as :class:`collections.abc.Set`
```